mmv1/templates/terraform/examples/big_query_routine_pyspark.tf.erb

```diff
@@ -0,0 +1,43 @@
+resource "google_bigquery_dataset" "test" {
```

Comment:
These examples will show up in the website documentation, so the spacing should be cleaned up a bit to be consistent. Same in the other example file.

Reply:
I overlooked the tab in spaces. Now it should clean.